prevodach.js

```diff
@@ -0,0 +1,48 @@
+const axios = require('axios').default;
+const { v4: uuidv4 } = require('uuid');
+
+// PLEASE DONT STEAL THIS
+// GCP SECRET VALUE ?
+var subscriptionKey = process.env['translator-key'];
```

Comment:
Hugo has a gitignored config.json where we could add this key to instead. `const { translatorKey } = require('config.json')` which we could use instead (and then PM hugo for a key), to avoid introducing new concepts. 

I find this way (gitignored config file) to be same debt in setup, but easier long term maintenance? In most circumstances I'd argue for env key though

Reply:
You can also request the contents of the file, so you can run the program locally.

Reply:
I hide many secrets, and can hide many more.